RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py

```diff
@@ -6,7 +6,7 @@
 from RecoTracker.TkTrackingRegions.GlobalTrackingRegion_cfi import *
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
 
-layerInfo = cms.PSet(
+layerInfo = dict(
```

Comment:
Since you are cleaning, you could further align inside the PSet's in `layerInfo`